Model/Sync/Reset/Main.php

```diff
@@ -0,0 +1,160 @@
+<?php
+
+namespace Yotpo\Core\Model\Sync\Reset;
+
+use Magento\Framework\App\ResourceConnection;
+use Yotpo\Core\Model\Config as CoreConfig;
+use Yotpo\Core\Model\AbstractJobs;
+
+class Main
+{
+    const DELETE_LIMIT = 10000;
+
+    /**
+     * @var int
+     */
+    protected $storeId = 0;
+
+    /**
+     * @var CoreConfig
+     */
+    protected $coreConfig;
+
+    /**
+     * @var ResourceConnection
+     */
+    protected $resourceConnection;
+
+    /**
+     * @var AbstractJobs
+     */
+    protected $abstractJobs;
+
+    /**
+     * @var array <string>
+     */
+    protected $cronJobCodes = [''];
```

Comment:
This is misleading.
```suggestion
    protected $cronJobCodes = [];
```

On `removeCronJobs` you may check of `if (empty($jobCodes))` instead of `if (!$jobCodes)`.
The object is always an array.

Reply:
This is done on the first part. We don't need to replace !$jobCodes as it is same as empty($jobCodes)

Reply:
I don't mind not using the `empty` method.
Mostly concerned about initializing the array with an empty string.
please initialize it as an empty array without faulty data as a placeholder
```
protected $cronJobCodes = [];
```

Reply:
This is done